fedora/jellyfin.spec

```diff
@@ -12,14 +12,15 @@ Release:        1%{?dist}
 Summary:        The Free Software Media System
 License:        GPLv3
 URL:            https://jellyfin.org
-# Jellyfin Server tarball created by `make -f .copr/Makefile srpm`, real URL ends with `v%{version}.tar.gz`
+# Jellyfin Server tarball created by `make -f .copr/Makefile srpm`, real URL ends with `v%%{version}.tar.gz`
 Source0:        jellyfin-server-%{version}.tar.gz
```

Comment:
Was the additional % accidentally added?

Reply:
No.  It's an _escape_.  Macros get expanded, even inside comments which is typically not what is desired.  The RPM linter will complain about them.